src/ProjectTemplates/Directory.Build.props

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <!-- The .csproj files in this folder do not actually produce .dll's. They are only present to produce .nupkgs and enable Visual Studio support -->
     <IsProjectReferenceProvider>false</IsProjectReferenceProvider>
+    <BlazorDisableThrowNavigationException>true</BlazorDisableThrowNavigationException>
```

Comment:
This only causes it to happen on our project templates, not on the created projects.

For the created projects (rather than adding a `Directory.Build.props`) stick it into the csproj file. (the csproj.in file here)